model/feature_flags.go

```diff
@@ -98,13 +101,14 @@ func (f *FeatureFlags) SetDefaults() {
 	f.BoardsProduct = true
 	f.SendWelcomePost = true
 	f.PostPriority = true
-	f.PeopleProduct = false
+	f.PeopleProduct = true
```

Comment:
Why do we need this?

Reply:
Ooops, that got left in from something I was working on for Open source Friday